pipeline.libsonnet

```diff
@@ -99,7 +99,7 @@
   },
 
   inputArtifact(id):: {
```

Comment:
we should rename the argument as well, shouldn’t we?

Reply:
yes, I missed this one in my PR, should I update my PR?